include/zephyr/libsbc/sbc.h

```diff
@@ -15,99 +18,119 @@
 #include "oi_status.h"
 #endif
 
+/** @brief SBC channel mode */
 enum __packed sbc_ch_mode {
 	SBC_CH_MODE_MONO,
 	SBC_CH_MODE_DUAL_CHANNEL,
 	SBC_CH_MODE_STEREO,
 	SBC_CH_MODE_JOINT_STEREO,
 };
 
+/** @brief SBC allocation method */
 enum __packed sbc_alloc_mthd {
 	SBC_ALLOC_MTHD_LOUDNESS,
 	SBC_ALLOC_MTHD_SNR,
 };
 
 #if defined(CONFIG_LIBSBC_ENCODER)
```

Comment:
unless I am missing something, please remove all these `#if defined(CONFIG_LIBSBC_ENCODER)` from this file (right now, among other things, this is hiding the headers to Doxygen - see https://builds.zephyrproject.io/zephyr/pr/95137/api-coverage/zephyr/libsbc/sbc.h.gcov.html)